support-frontend/assets/components/paymentFrequencyButtons/paymentFrequencyButtons.tsx

```diff
@@ -2,9 +2,11 @@ import type { SerializedStyles } from '@emotion/react';
 import { css } from '@emotion/react';
 import { from, palette, space, textSans } from '@guardian/source-foundations';
 import { useState } from 'react';
+import type { RegularContributionType } from 'helpers/contributions';
 
 interface PaymentFrequencyButtonObj {
 	label: string;
```

Comment:
I'd be inclined to rename `paymentFrequency` to `paymentFrequencyID`, as it's a bit more clear what this prop does (it's used as an ID), I'd maybe rename `label` to `paymentFrequencyLabel` too.